lib/DBIx/Class/ResultSource.pm

```diff
@@ -724,6 +725,11 @@ This will result in a unique constraint named
 C<table_column1_column2>, where C<table> is replaced with the table
 name.
 
+The "extra" hashref will be passed to
```

Comment:
It's not an `extra hashref` anymore. User specifies explicit `uc_name => { extra => <what will go to SQLT>, <other options> }`. This way you keep the API extensible in the future, instead of locking things into *all of this will be passed to SQLT and you better hope it interoperates forever, KTHXBAI!*

In fact I would call it `sqlt_extra` (even though there is precedent in https://metacpan.org/pod/DBIx::Class::ResultSource#extra, but that's arguably bad precedent)